klog.go

```diff
@@ -698,7 +698,7 @@ func (buf *buffer) someDigits(i, d int) int {
 	return copy(buf.tmp[i:], buf.tmp[j:])
 }
 
-func (l *loggingT) println(s severity, logr logr.Logger, filter LogFilter, args ...interface{}) {
+func (l *loggingT) println(s severity, logr *logr.Logger, filter LogFilter, args ...interface{}) {
```

Comment:
Using "logr" as parameter name is awkward because is shadows the package name. The logr.go example uses "logger" as variable name - should we also use that here?

I understand that this PR is about making other changes, but as you seem to touch most of the lines where `logr` occurs, perhaps now would be a good opportunity to also change the name.

Reply:
on it.